{"@t":"2020-01-01T00:00:00.0000000Z","@m":"Handling message on in-process sender with payload {\"Payload\":10} (Message ID: test-message-id, Trace ID: test-trace-id)","@i":"hash","TransportTypeName":"in-process","TransportRole":"sender","MessagePayload":"{\"Payload\":10}","MessageId":"test-message-id","TraceId":"test-trace-id","EventId":{"Id":711195907,"Name":"conqueror-message"},"SourceContext":"CustomCategory_TestMessageWithoutResponse"}
{"@t":"2020-01-01T00:00:00.0000000Z","@m":"Handling message with payload {\"Payload\":10} (Message ID: test-message-id, Trace ID: test-trace-id)","@i":"hash","MessagePayload":"{\"Payload\":10}","MessageId":"test-message-id","TraceId":"test-trace-id","EventId":{"Id":711195907,"Name":"conqueror-message"},"SourceContext":"CustomCategory_TestMessageWithoutResponse"}
{"@t":"2020-01-01T00:00:00.0000000Z","@m":"An exception occurred while handling message after 123.4560ms (Message ID: test-message-id, Trace ID: test-trace-id)","@i":"hash","@l":"Error","@x":"Conqueror.Middleware.Logging.Tests.Messaging.LoggingMiddlewareTestMessages+TestException: test exception\n   at Conqueror.Middleware.Logging.Tests.Messaging.LoggingMiddlewareTestMessages.TestMessageWithoutResponseHandler.Handle(TestMessageWithoutResponse message, CancellationToken cancellationToken) in {ProjectDirectory}Messaging/LoggingMiddlewareTestMessages.cs:line FIXED\n   at Conqueror.Middleware.Logging.Tests.Messaging.LoggingMiddlewareTestMessages.TestMessageWithoutResponse.IHandler.global::Conqueror.IMessageHandler<Conqueror.Middleware.Logging.Tests.Messaging.LoggingMiddlewareTestMessages.TestMessageWithoutResponse,Conqueror.UnitMessageResponse,Conqueror.Middleware.Logging.Tests.Messaging.LoggingMiddlewareTestMessages.TestMessageWithoutResponse.IHandler>.Invoke(IHandler handler, TestMessageWithoutResponse message, CancellationToken cancellationToken) in {ProjectDirectory}obj/Debug/net8.0/Conqueror.SourceGenerators/Conqueror.SourceGenerators.Messaging.MessageTypeGenerator/Conqueror.Middleware.Logging.Tests.Messaging.LoggingMiddlewareTestMessages.TestMessageWithoutResponse_ConquerorMessageType.g.cs:line FIXED\n   at Conqueror.Middleware.Logging.Messaging.LoggingMessageMiddleware`2.Execute(MessageMiddlewareContext`2 ctx) in {SolutionDirectory}src/Conqueror.Middleware.Logging/Messaging/LoggingMessageMiddleware.cs:line FIXED\n   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine)\n   at Conqueror.Middleware.Logging.Messaging.LoggingMessageMiddleware`2.Execute(MessageMiddlewareContext`2 ctx)","ResponseLatency":123.456,"MessageId":"test-message-id","TraceId":"test-trace-id","EventId":{"Id":2027292319,"Name":"conqueror-message-exception"},"SourceContext":"CustomCategory_TestMessageWithoutResponse"}
{"@t":"2020-01-01T00:00:00.0000000Z","@m":"An exception occurred while handling message on in-process sender after 123.4560ms (Message ID: test-message-id, Trace ID: test-trace-id)","@i":"hash","@l":"Error","@x":"Conqueror.Middleware.Logging.Tests.Messaging.LoggingMiddlewareTestMessages+TestException: test exception\n   at Conqueror.Middleware.Logging.Tests.Messaging.LoggingMiddlewareTestMessages.TestMessageWithoutResponseHandler.Handle(TestMessageWithoutResponse message, CancellationToken cancellationToken) in {ProjectDirectory}Messaging/LoggingMiddlewareTestMessages.cs:line FIXED\n   at Conqueror.Middleware.Logging.Tests.Messaging.LoggingMiddlewareTestMessages.TestMessageWithoutResponse.IHandler.global::Conqueror.IMessageHandler<Conqueror.Middleware.Logging.Tests.Messaging.LoggingMiddlewareTestMessages.TestMessageWithoutResponse,Conqueror.UnitMessageResponse,Conqueror.Middleware.Logging.Tests.Messaging.LoggingMiddlewareTestMessages.TestMessageWithoutResponse.IHandler>.Invoke(IHandler handler, TestMessageWithoutResponse message, CancellationToken cancellationToken) in {ProjectDirectory}obj/Debug/net8.0/Conqueror.SourceGenerators/Conqueror.SourceGenerators.Messaging.MessageTypeGenerator/Conqueror.Middleware.Logging.Tests.Messaging.LoggingMiddlewareTestMessages.TestMessageWithoutResponse_ConquerorMessageType.g.cs:line FIXED\n   at Conqueror.Middleware.Logging.Messaging.LoggingMessageMiddleware`2.Execute(MessageMiddlewareContext`2 ctx) in {SolutionDirectory}src/Conqueror.Middleware.Logging/Messaging/LoggingMessageMiddleware.cs:line FIXED\n   at Conqueror.Messaging.MessagePipelineRunner`2.Execute(IServiceProvider serviceProvider, TMessage initialMessage, IMessageSender`2 sender, MessageTransportType transportType, CancellationToken cancellationToken) in {SolutionDirectory}src/Conqueror/Messaging/MessagePipelineRunner.cs:line FIXED\n   at Conqueror.Messaging.MessageDispatcher`2.Dispatch(TMessage message, CancellationToken cancellationToken) in {SolutionDirectory}src/Conqueror/Messaging/MessageDispatcher.cs:line FIXED","TransportTypeName":"in-process","TransportRole":"sender","ResponseLatency":123.456,"MessageId":"test-message-id","TraceId":"test-trace-id","EventId":{"Id":2027292319,"Name":"conqueror-message-exception"},"SourceContext":"CustomCategory_TestMessageWithoutResponse"}
